libr/core/disasm.c

```diff
@@ -1009,6 +1009,9 @@ static void ds_build_op_str(RDisasmState *ds, bool print_color) {
 	core->parser->relsub = r_config_get_i (core->config, "asm.relsub");
 	core->parser->regsub = r_config_get_i (core->config, "asm.regsub");
 	core->parser->relsub_addr = 0;
+	if (core->parser->relsub && ds->analop.type == R_ANAL_OP_TYPE_LEA && ds->analop.ptr != UT64_MAX) {
+		core->parser->relsub_addr = ds->analop.ptr;
```

Comment:
Is this still necessary after the change in anal?

Reply:
The assignment to `core->parser->relsub_addr` still needs to be done.